lib/tape/installer.rb

```diff
@@ -42,25 +56,25 @@ def rails_app?
     end
 
     def copy_static_app_examples
-      copy_example 'templates/static_html/omnibox.example.yml', 'omnibox.yml'
-      copy_example 'templates/static_html/deploy.example.yml', 'deploy.yml'
-      copy_example 'templates/static_html/tape_vars.example.yml', 'tape_vars.yml'
+      copy_example 'templates/static_html/omnibox.example.yml', "#{tapefiles_dir}/omnibox.yml"
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.